### PR TITLE
1059 padding margin accordions

### LINKED ISSUE #1059: Fix padding/margins on accordions
### Overview

There is a padding  issue on the accordions that is causing too much space at the bottom of test in the drop down. this appears to be 

### Action Items

- [x] identify which Accordions have this space 
- [x] determine source of issue (html coming from the database?)
- [x] make margins and padding consistent on all accordions.

### PR DESCRIPTION
**Pull request by jahpe777**
Fixed padding/margins on accordions

Closes #1059

### CODE REVIEW

File: server/db/migration/V20220413.2005__adjust_tooltips_1059.sql [hunk 1]
```diff
@@ -0,0 +1,112 @@
+update CalculationRule set 
+choices = '[{"id": "", "name": "N/A"},
+{"id": "1", "name": "20% of State Density Bonus"},
+{"id": "2", "name": "TOC Tiers 1-3"},
+{"id": "3", "name": "TOC Tier 4"},
+{"id": "4", "name": "100% Affordable"}]',
```
Comment (fyliu) [inline server/db/migration/V20220413.2005__adjust_tooltips_1059.sql:6]: These lines are not necessary to update the description. They can be safely deleted.
```suggestion
```

File: server/db/migration/V20220413.2005__adjust_tooltips_1059.sql [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+where calculationId = 1 and code = 'STRATEGY_SHARED_MOBILITY_2'
+
+update CalculationRule set 
+ description = 'Provide funding to a local transit provider for 
+ improvements that improve service quality (reduce headways, etc.) 
+ at transit stops within ¼ mile radius of the project site. Funds 
+ could also contribute to an existing shuttle or microtransit 
+ service (e.g. DASH) in consultation with LADOT if this option 
+ is available near the project site.'
+where calculationId = 1 and code = 'STRATEGY_TRANSIT_ACCESS_4'
```
Comment (fyliu) [inline server/db/migration/V20220413.2005__adjust_tooltips_1059.sql:112]: These lines are extras from V20210526.1439__adjust_tooltips_827_832.sql. They can be safely deleted.
```suggestion
```

File: server/db/migration/V20220413.1837__rule_adjustments_1059.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+	description = '<div><p>The City defines "habitable rooms" as enclosed spaces used for living purposes. 
+	This includes bedrooms, living rooms and dining rooms, given that they are all enclosed.
+	</p>
+	<p>Any non-enclosed alcove more than 50 square feet in size is also included in this definition 
+	(dining area excluded). The City''s definition of habitable room excludes lobbies, hallways, 
+	and bathrooms. The City includes kitchens in the definition only for calculating parking requirements.
+	</p>
+	<p style="padding-bottom:0;">Therefore, a living room, dining room, and kitchen, all completely separated from each 
+	other with walls, will count as three habitable rooms for parking calculation purposes.</p></div>'
+WHERE calculationId = 1 and code in ('UNITS_HABIT_LT3', 'UNITS_HABIT_3', 'UNITS_HABIT_GT3')
```
Comment (fyliu) [inline server/db/migration/V20220413.1837__rule_adjustments_1059.sql:11]: This is a duplicate file with padding-bottom:0 that'll be overwritten later with the margin-bottom:0 version
```suggestion
```

File: server/db/migration/V20220413.1856__adjust_tooltips_1059.sql [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+where calculationId = 1 and code = 'STRATEGY_SHARED_MOBILITY_2'
+
+update CalculationRule set 
+ description = 'Provide funding to a local transit provider for 
+ improvements that improve service quality (reduce headways, etc.) 
+ at transit stops within ¼ mile radius of the project site. Funds 
+ could also contribute to an existing shuttle or microtransit 
+ service (e.g. DASH) in consultation with LADOT if this option 
+ is available near the project site.'
+where calculationId = 1 and code = 'STRATEGY_TRANSIT_ACCESS_4'
```
Comment (fyliu) [inline server/db/migration/V20220413.1856__adjust_tooltips_1059.sql:112]: This is a duplicate file with padding-bottom:0 that'll be overwritten later with the margin-bottom:0 version. This file can be safely deleted.
```suggestion
```

File: server/db/migration/V20220413.1922__consolidate_bike_share_strategies_1059.sql [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+UPDATE CalculationRule SET
+functionBody = 'return <<PTS_BIKE_1>> + <<PTS_BIKE_3>> + <<PTS_BIKE_4>> + <<PTS_BIKE_5>> + <<PTS_BIKE_BONUS>>;'
+where calculationId =1 and code = 'PTS_BIKE'
+
+DELETE CalculationRule 
+where calculationId =1 and code = 'STRATEGY_BIKE_2'
+
+DELETE CalculationRule 
+where calculationId =1 and code = 'PTS_BIKE_2'
+
```
Comment (fyliu) [inline server/db/migration/V20220413.1922__consolidate_bike_share_strategies_1059.sql:35]: This is a duplicate file with padding-bottom:0 that'll be overwritten later with the margin-bottom:0 version. This file can be safely deleted.
```suggestion
```

**Review by fyliu — CHANGES_REQUESTED** [review]
The fixes look great!
There's some extra lines and files that should probably be deleted.

**Comment by jahpe777** [comment]
Updated sql files based on review by @fyliu

**Comment by jahpe777** [comment]
made edits based on Fangs comments for encouragement program strategy

**Review by fyliu — APPROVED** [review]
Looks good. Thanks for making the changes.

**Pull request merged**